scenarios/erc20_multichain_factory/test/TestDeleteEntity.res

```diff
@@ -2,6 +2,19 @@ open Belt
 open RescriptMocha
 
 let config = RegisterHandlers.getConfig()
+// Keep only the first chain
+let config = Config.make(
+  ~shouldRollbackOnReorg=false,
+  ~shouldSaveFullHistory=false,
+  ~isUnorderedMultichainMode=false,
+  ~chains=config.chainMap
+  ->ChainMap.entries
+  ->Array.keepMap(((chain, config)) =>
+    chain == RollbackMultichain_test.Mock.Chain1.chain ? Some(config) : None
+  ),
+  ~enableRawEvents=false,
+  ~entities=config.entities->Obj.magic,
+)
```

Comment:
This one and a few other tests randomly broke after the fix. It is caused by them having `shouldRollbackOnReorg` enabled. The chain we are testing gets some height and considered to be in the reorg threshold, while other chains stay at block 0 and have in reorg threshold as false. To fix this I remove other chains from the tests which we don't use.

Reply:
Ok cool 👍🏼